packages/extension-ui/src/Popup/Accounts/index.tsx

```diff
@@ -42,6 +43,48 @@ function Accounts({ className }: Props): React.ReactElement {
     setFilter(filter.toLowerCase());
   }, []);
 
+  const accountsByGenesisHash: { [key: string]: AccountWithChildren[] } = filteredAccount.reduce(
+    (result: { [key: string]: AccountWithChildren[] }, account) => {
+      const { genesisHash } = account;
+      const foundKey = Object.keys(knownGenesis).find((key) => knownGenesis[key].includes(genesisHash ?? ''));
+
+      if (!foundKey) {
+        if (!result.any) {
+          result.any = [];
+        }
```

Comment:
To avoid nested if's I would provide {any: []} in the default value at the bottom so it's always a part of the result.